installer/default_platform.conf

```diff
@@ -434,14 +434,23 @@ bootloader_menu_config()
         ${onie_bin} onie-support /tmp
         mv $onie_initrd_tmp/tmp/onie-support*.tar.bz2 $demo_mnt/$image_dir/
 
+        echo "firmware=$firmware"
         if [ "$firmware" = "uefi" ] ; then
-            secure_boot_state=$(mokutil --sb-state)
+            secure_boot_state=0
+            reg_sb_guid=""
+            ENABLED=1
+            echo "checking secure boot state"
+            reg_sb_guid=$(efivar -l | grep "SecureBoot") || echo "Secure Boot GUID not found in efivar list"
```

Comment:
This condition will fail when there are multiple efi entry whose efi variable name start with  "SecureBoot"
Ex:
ONIE:/tmp # efivar -l | grep "SecureBoot"
59d1c24f-50f1-401a-b101-f33e0daed443-SecureBootEnforce
8be4df61-93ca-11d2-aa0d-00e098032b8c-SecureBoot

Can you fix this ?
efivar -l | grep "SecureBoot$"

Reply:
sure I will, 
BTW do you know what its "SecureBootEnforce" use case?